lib/elixir/lib/application.ex

```diff
@@ -314,6 +314,7 @@ defmodule Application do
   application environment. `new` is a keyword list with all new keys
   and their values. `removed` is a list with all removed keys.
   """
+  @doc since: "1.8.0"
```

Comment:
The callback was always there but not documented, so I think we can remove this one. :)